mopidy/Dockerfile

```diff
@@ -0,0 +1,49 @@
+FROM debian:stretch-slim
+
```

Comment:
Is there a `debian:buster-slim` ?

Reply:
👍 Have updated to buster and all still working. Latest and greatest.